src/cli/pytest_commands/processors.py

```diff
@@ -92,8 +96,12 @@ def process_args(self, args: List[str]) -> List[str]:
         if os.getenv("HIVE_LOGLEVEL") is not None:
             warnings.warn("HIVE_LOG_LEVEL is not yet supported.", stacklevel=2)
 
-        modified_args.extend(["-p", "pytest_plugins.pytest_hive.pytest_hive"])
-
+        if self.command_name == "engine":
+            modified_args.extend(["-p", "pytest_plugins.consume.simulators.engine.conftest"])
+        elif self.command_name == "rlp":
+            modified_args.extend(["-p", "pytest_plugins.consume.simulators.rlp.conftest"])
+        else:
+            raise ValueError(f"Unknown command name: {self.command_name}")
```

Comment:
With this PR, we now explicitly register a single plugin for each consume sub-command; the plugin specified here dynamically registers all the required plugins in order to define the sub-command behavior. This is achieved by defining a `pytest_plugins` variable, e.g.,
https://github.com/ethereum/execution-spec-tests/blob/b2e4a3bc087f8d0c0764d22c080207e3822d9e20/src/pytest_plugins/consume/simulators/engine/conftest.py#L17-L24

Reply:
Didn't even knew this was possible!

Link for future reference: https://docs.pytest.org/en/stable/reference/reference.html#globalvar-pytest_plugins

Reply:
Thanks! Improved the description with the above comment and added the link!